backend/internal/model/game.go

```diff
@@ -2,7 +2,17 @@ package model
 
 import "errors"
 
-type Game struct {
+const (
+	teamWhite int = 1
+	teamBlue  int = 2
+)
+
+type Game interface {
```

Comment:
I'd define this interface where it's used - next to the server, and I would have left the original model how it was, but this is fine too